config/samples/banzaicloud_v1beta1_kafkacluster.yaml

```diff
@@ -316,8 +316,11 @@ spec:
     # sslSecrets contains information about ssl related kubernetes secrets if one of the
     # listener setting type set to ssl these fields must be populated too.
     sslSecrets:
-      # tlsSecretName should contain all ssl certs required by kafka including: caCert, caKey, clientCert, clientKey
-      # serverCert, serverKey, peerCert, peerKey
+      # when create is false then the user has to provide the TLS certificate(s) and private key in PEM format in the refererenced secret below.
+      # this certificate and private key is used by cert-manager to issue client and server certificates. The client certificate is used by the Koperator , Cruise Control, 
+      # and Cruise Control Metrics Reporter to communicate Kafka brokers using listener with SSL enabled.
```

Comment:
```suggestion
      # and Cruise Control Metrics Reporter to communicate with Kafka brokers using listener(s) with SSL enabled.
```

Reply:
It communicates with one listener which is I think the controller